src/WizardLayout/WizardLayout.less

```diff
@@ -89,6 +89,7 @@
   .text--large();
   .text--semi-bold();
   .margin--bottom--none();
+  .padding--y--xs();
```

Comment:
Does padding match the default padding of the <p> tag out of curiosity?

Reply:
I think we should follow what [a recent PR](https://github.com/Clever/components/pull/833) did that's almost identical to the changes here: They conditionally render a `<p>` or `<div>` tag depending on if the prop is a string or a React node

Reply:
This pr is a little different because it handles the subtitle of the entire wizard which makes changes between paddings inconspicuous. In mine, it is a step in a wizard and when stepping through small differences can be noticed fairly well. This is why I wanted to make them all have the same padding but it is a little different than the default. I can try further to make them match so we can do the render conditionally without drawing attention when going to the next step.

Reply:
ah, okay, makes sense. I defer to Ben's question then!

Reply:
@brollin now it is the same. it was due to the line-height property of the `<p>` tag.

Reply:
Seems reasonable to me! 👍